drivers/timer/stm32_lptim_timer.c

```diff
@@ -204,9 +278,9 @@ void sys_clock_set_timeout(int32_t ticks, bool idle)
 	next_arr = (((lp_time * CONFIG_SYS_CLOCK_TICKS_PER_SEC)
 			/ lptim_clock_freq) + 1) * lptim_clock_freq
 			/ (CONFIG_SYS_CLOCK_TICKS_PER_SEC);
-	/* add count unit from the expected nb of Ticks */
-	next_arr = next_arr + ((uint32_t)(ticks) * lptim_clock_freq)
-			/ CONFIG_SYS_CLOCK_TICKS_PER_SEC - 1;
+	next_arr = next_arr + ((uint32_t)(ticks) * LPTIM_CLOCK)
```

Comment:
```suggestion
	next_arr = next_arr + ((uint32_t)(ticks) * lptim_clock_freq)
```

Reply:
_lptim_clock_freq/LPTIM_CLOCK_RATIO_ indeed in that formula